examples/optimizer.py

```diff
@@ -2,8 +2,9 @@
 from transformers import AdamW
 
 def initialize_optimizer(config, model):
-    if config.model.startswith('bert'):
-        assert config.optimizer=='AdamW', 'Only AdamW supported for BERT models'
+    if 'bert' in config.model:
```

Comment:
CC @ssagawa 

Should we just remove this restriction? If people want to change optimizer for BERT models, that seems ok?